src/app/project-detail/project-detail.component.spec.ts

```diff
@@ -200,52 +251,28 @@ describe('ProjectDetailComponent', () => {
   });
 
   describe('setDocumentUrl', () => {
+    beforeEach(() => {
+      component.setDocumentUrl();
+    });
+
     it('should set results.documentUrl to \'\' when no document url ', () => {
-      const data = {
-        recent_activities: [
-          {
-            documentUrl: ''
-          }
-        ]
-      };
-      component.setDocumentUrl(data);
-      expect(data.recent_activities[0].documentUrl).toBe('');
+      expect(component.news[0].documentUrl).toBe('');
     });
 
     it('should not change results.documentUrl when given a www url', () => {
-      const data = {
-        recent_activities: [
-          {
-            documentUrl: 'http://www.test.com'
-          }
-        ]
-      };
-      component.setDocumentUrl(data);
-      expect(data.recent_activities[0].documentUrl).toBe('http://www.test.com');
+      expect(component.news[1].documentUrl).toBe('http://www.test.com');
     });
 
     it('should set results.documentUrl to \'http://localhost:3000/blarg\' when given an esm-server document ', () => {
-      const data = {
-        recent_activities: [
-          {
-            documentUrl: '/blarg'
-          }
-        ]
-      };
-      component.setDocumentUrl(data);
-      expect(data.recent_activities[0].documentUrl).toBe('http://localhost:3000/blarg');
+      expect(component.news[2].documentUrl).toBe('http://localhost:3000/blarg');
     });
   });
 
   describe('getDisplayedElementCountMessage', () => {
     beforeEach(() => {
-      component.project = new Project();
-      component.project.recent_activities = [
-        new News({ headline: 'Big mine' , type: 'news' }),
-        new News({ headline: 'Medium', type: 'public comment period' }),
-        new News({ headline: 'Bigger mine', type: 'public comment period' }),
-        new News({ headline: 'Small', type: 'news' })
-      ];
+      this.filter = undefined;
+      this.NewsTypeFilter = undefined;
```

Comment:
Does this do anything? Do we need to call 'component.NewsTypeFilter..."?

Reply:
For some reason I meant to reset each filter (so ```component.filterName = undefined```) because it seemed to be required for my tests to pass. it looks like ti was a fluke, so I am removing it.